lib/fixme.ex

```diff
@@ -2,20 +2,26 @@ defmodule FIXME do
   defmacro fixme(
     {:-, _, [{:-, _, [year, month]}, day]},
     message,
-    opts \\ :default_opts
+    opts \\ []
   ) do
     {opts, _} = Code.eval_quoted(opts)
 
     # Injectable "today" for tests.
-    current_date = case opts do
-      :default_opts -> :calendar.local_time |> elem(0)
-      [today: date] -> date
-    end
+    current_date = opts[:today] || (:calendar.local_time |> elem(0))
+
+    # Opt into compile time warnings
+    warn = opts[:warn]
```

Comment:
@nimish-mehta Thanks so much for making that change! I just wanted to let you know I haven't had the time to look at your changes since I've been extra busy the last few days. I will be for a few more days – will look after that!